mealpy/mealpy.py

```diff
@@ -64,6 +64,7 @@ def login(self):
     @staticmethod
     def get_cities():
         request = requests.post(CITIES_URL, headers=HEADERS)
+        request.raise_for_status()
```

Comment:
This will raise error and cause script to fail. I guess we should consider handling that gracefully and hook it into the retry logic. That's out of scope for this PR though, it's existing behaviour.

Reply:
Agree that we should handle the exceptions in a separate PR, and do this for other API methods too.